src/test/resources/application-test.conf

```diff
@@ -22,6 +22,7 @@ agreement-process {
     party-process                  = "party-process"
   }
 
+  events-queue-name         = "events-queue-name"
```

Comment:
[same](https://github.com/pagopa/interop-be-agreement-process/pull/238/files#r1253046683)